### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Added responsiveness
- Updated readme

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]
UI looks Great.
No issues found, you can merge it.

File: src/components/TaskCard/task-card.scss [hunk 1]
```diff
@@ -12,7 +14,7 @@
   align-items: center;
 
   &__text-container{
-    flex: 1;
+    // flex: 1 1;
```
Comment (shashikantmaurya0007) [inline src/components/TaskCard/task-card.scss]: remove this comment

**Review by shashikantmaurya0007 — APPROVED** [review]
as usual your ui is awesome:)

File: src/components/TaskCard/task-card.scss (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/TaskCard/task-card.scss]: removed it

**Pull request merged**